.github/crystal-spec.json

```
@@ -0,0 +1,59 @@
+{
+    "problemMatcher": [
+        {
+            "owner": "crystal-spec-multi-line",
+            "pattern": [
+                {
+                    "regexp": "^ {0,2}\\d+\\) (.+)$"
+                },
+                {
+                    "regexp": "^ {5}(.+)$"
+                },
+                {
+                    "regexp": "^$"
+                },
+                {
+                    "regexp": "^ {5}\\s*(.+)$"
+                },
+                {
+                    "regexp": "^ {5}\\s*(.+)$",
+                    "message": 1
+                },
+                {
+                    "regexp": "^$"
+                },
+                {
+                    "regexp": "^ {5}# ([^\\s]+):(\\d+)$",
```

I suppose paths could technically contain white space. In `crystal.json` the file name pattern is just `.+` as well.

```suggestion
                    "regexp": "^ {5}# (.+):(\\d+)$",
```